docs/user_guide/sql-ai-shell.md

```diff
@@ -87,3 +104,56 @@ You can disable the AI shell entirely by adding a flag to either your personal o
 {% ini-include section="plugins.bks-ai-shell" %}
 
 Even if the application user has previously entered an API key, this setting will disable the feature and lock out any further usage.
+
+### Troubleshooting
```

Comment:
Troubleshooting should not be nested under 'privacy'. This should be h2, not h3.